src/main/java/sg/ncl/MainController.java

```diff
@@ -148,6 +152,11 @@ public String calendar1() {
         return "calendar1";
     }
 
+    @RequestMapping("/tools")
+    public String tools() {
+        return "tools";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this method and declare a constant for this value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3400)